src/DependencyInjection/PimcoreDataHubExtension.php

```diff
@@ -52,29 +52,18 @@ public function prepend(ContainerBuilder $container)
             $loader->load('doctrine_migrations.yml');
         }
 
-        $configDir = PIMCORE_CONFIGURATION_DIRECTORY . '/data_hub';
-
-        if (\Pimcore\Version::getMajorVersion() >= 11) {
-            $containerConfig = ConfigurationHelper::getConfigNodeFromSymfonyTree($container, 'pimcore_data_hub');
-            $configDir = $containerConfig['config_location']['data_hub']['write_target']['options']['directory'];
-        }
+        $containerConfig = ConfigurationHelper::getConfigNodeFromSymfonyTree($container, 'pimcore_data_hub');
+        $configDir = $containerConfig['config_location']['data_hub']['write_target']['options']['directory'];
 
         $configLoader = new YamlFileLoader(
             $container,
             new FileLocator($configDir)
         );
 
         //load datahub configs
-        if (\Pimcore\Version::getMajorVersion() >= 11) {
-            $configs = ConfigurationHelper::getSymfonyConfigFiles($configDir);
-            foreach ($configs as $config) {
-                $configLoader->load($config);
-            }
-        } else {
-            $configLocator = new \Pimcore\Bundle\DataHubBundle\Configuration\DatahubConfigLocator();
-            foreach ($configLocator->locate('config') as $config) {
-                $configLoader->load($config);
-            }
+        $configs = ConfigurationHelper::getSymfonyConfigFiles($configDir);
```

Comment:
so now this only considers the directory from the Symfony config not the default [dir ](https://github.com/pimcore/data-hub/pull/784/files#diff-d3cd915a931a9be0a3278d79a468a433bd6016436b5a82f9188e72181d3170c5L55) anymore. what happens now if i use `PIMCORE_CONFIGURATION_DIRECTORY`?

Reply:
@dvesh3 Did you mean suppose if we didn't give any write_target in the yaml configuration, what will be the value of $configDir ? In that case, it will be set to '/var/config/data_hub'. https://github.com/pimcore/data-hub/blob/e28c7c4523367efa768e34aa86cb2266cc1468c0/src/DependencyInjection/Configuration.php#L52

Reply:
That's right, so I will fix this. Thanks for finding this